# --- T2-COPYRIGHT-BEGIN ---
# t2/architecture/cris/package/*/hotfix-cris-no-atomic-libfunc.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- gcc-9.3.0/gcc/config/cris/cris.opt.vanilla	2025-07-18 23:56:41.228957852 +0200
+++ gcc-9.3.0/gcc/config/cris/cris.opt	2025-07-18 23:56:49.035322022 +0200
@@ -184,7 +184,7 @@
 Emit checks causing \"break 8\" instructions to execute when applying atomic builtins on misaligned memory.
 
 munaligned-atomic-may-use-library
-Target Report Var(cris_atomics_calling_libfunc) Init(2)
+Target Report Var(cris_atomics_calling_libfunc) Init(0)
 Handle atomic builtins that may be applied to unaligned data by calling library functions. Overrides -mtrap-unaligned-atomic.
 
 ; TARGET_SVINTO: Currently this just affects alignment.  FIXME:
